src/Api/Serializer/RestSerializer.php

```diff
@@ -127,6 +127,12 @@ private function applyHeader($name, Shape $member, $value, array &$opts)
             $value = TimestampShape::format($value, 'rfc822');
         }
 
+        if (!empty($member->offsetGet('jsonvalue'))
+            && $member->offsetGet('jsonvalue')
+        ) {
+            $value = base64_encode(json_encode(json_decode($value)));
```

Comment:
The same reason described above. This I found to be a concise way of removing any new line characters etc.